src/osdc/Objecter.cc

```diff
@@ -3248,6 +3245,9 @@ void Objecter::_send_op(Op *op, MOSDOp *m)
     m->trace.init("op msg", nullptr, &op->trace);
   }
   op->session->con->send_message(m);
+
+  logger->inc(l_osdc_op_send);
+  logger->inc(l_osdc_op_send_bytes, m->get_data().length());
```

Comment:
i don't think `MOSDOp` has a non-zeo `data` payload. `MCommand` does. could you point me where `set_data()` is called for the `MOSDOp`?